yara.ungram

```diff
@@ -6,9 +6,21 @@ Rule =
 
 BlockExpr = 
     '{'
-        Strings? Condition
+        Meta? Strings? Condition
     '}'
 
+Meta = 
+    'meta' ':'
+        MetaStmt*
+
+MetaStmt =
+    'identifier' '=' 
+        |  'true'
```

Comment:
It seems a bit inconsistent to have other types of literals, but for bool have values explicitly listed here.

I am not pushing to change it, but have you considered to have `'bool_lit'` here?

Reply:
Yes, thank you! This is from subset where there was no point in trying to make it consistant as there were no 'string_lit' or 'int_lit' values. I will unify it.

Reply:
Fixed in latest commit together with hex string support. Thank you again for a review